store/tikv/client.go

```diff
@@ -62,7 +62,6 @@ func newRPCClient() *rpcClient {
 
 // SendCopReq sends a Request to co-processor and receives Response.
 func (c *rpcClient) SendCopReq(addr string, req *coprocessor.Request, timeout time.Duration) (*coprocessor.Response, error) {
-	sendReqCounter.WithLabelValues("cop").Inc()
```

Comment:
We already have a `sendReqHistogram` which creates a counter implicitly, so this counter is redundant.